packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -94,233 +101,194 @@ export const treeItemStyles: (
     definition: TreeItemOptions
 ) => ElementStyles = (context: ElementDefinitionContext, definition: TreeItemOptions) =>
     css`
-    ${display("block")} :host {
-        contain: content;
-        position: relative;
-        outline: none;
-        color: ${neutralForegroundRest};
-        background: ${neutralFillStealthRest};
-        cursor: pointer;
-        font-family: ${bodyFont};
-        --expand-collapse-button-size: calc(${heightNumber} * 1px);
-        --tree-item-nested-width: 0;
-    }
+        ${display("block")} :host {
+            --expand-collapse-button-size: calc(${heightNumber} * 1px);
+            --tree-item-nested-width: 0;
+            contain: content;
+            position: relative;
+            outline: none;
+            color: ${neutralForegroundRest};
+            fill: currentcolor;
+            cursor: pointer;
+            font-family: ${bodyFont};
+        }
 
-    :host(:focus) > .positioning-region {
-        outline: none;
-    }
+        :host(:focus) > .positioning-region,
+        :host(:focus) .content-region {
+            outline: none;
+        }
 
-    :host(:focus) .content-region {
-        outline: none;
-    }
+        .positioning-region {
+            display: flex;
+            position: relative;
+            box-sizing: border-box;
+            background: ${neutralFillStealthRest};
+            border: calc(${strokeWidth} * 1px) solid transparent;
+            border-radius: calc(${controlCornerRadius} * 1px);
+            height: calc((${heightNumber} + 1) * 1px);
+        }
 
-    :host(:${focusVisible}) .positioning-region {
-        border: ${focusStrokeOuter} calc(${strokeWidth} * 1px) solid;
-        border-radius: calc(${controlCornerRadius} * 1px);
-        color: ${neutralForegroundRest};
-    }
+        :host(:not([disabled])) .positioning-region:hover {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
 
-    .positioning-region {
-        display: flex;
-        position: relative;
-        box-sizing: border-box;
-        border: transparent calc(${strokeWidth} * 1px) solid;
-        height: calc((${heightNumber} + 1) * 1px);
-    }
+        :host(:not([disabled])) .positioning-region:active,
+        :host(:not([disabled])[selected]) .positioning-region {
```

Comment:
I don't know why my previous comment is no longer lined up, but what I intended regarding the use of the `active` state color is that it is literally only ever used for `:active` selection state. That's the intent behind the color recipes being semantic. Not that `active` happens to be the "right" color so we use it wherever it's needed.

The selection state should have its own separate recipe. For instance in Fluent this is `neutralSecondary`, which also has the full set of `rest`, `hover`, etc. states. A "selected" element can still receive interaction and should respond accordingly,

Also, please add the contrasting color outline we've added to the listbox option as well.

Reply:
I separated them and added the outline similar to the list box option